tools/topology/m4/demux_coef_default.m4

```diff
@@ -0,0 +1,11 @@
+CONTROLBYTES_PRIV(DEMUX_priv,
```

Comment:
How are these generated ? Which structure, tool is used ?

Reply:
@lgirdwood my private c-tool to "binarize" @akloniex mux config struct. I don't know what's the procedure: I can add it somewhere in sof, but it takes me some time to make it nice and generic.

Reply:
yes please, any tools should be upstreamed (usually as part of this PR), but this can be another PR this time.